charts/scalar-manager/templates/rolebinding.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  namespace: {{ .Release.Namespace }}
+  name: {{ include "scalar-manager.fullname" . }}
+subjects:
+- kind: ServiceAccount
+  name: {{ include "scalar-manager.serviceAccountName" . }}
+  apiGroup: ""
+roleRef:
+  kind: Role
+  name: {{ include "scalar-manager.fullname" . }}
+  apiGroup: ""
```

Comment:
```suggestion
  apiGroup: rbac.authorization.k8s.io
```
Maybe, this configuration also based on my suggestion before, but I think the correct value of `roleRef.apiGroup` is `rbac.authorization.k8s.io` when we set `Role` to the `roleRef.kind`.

We can confirm the `apiGroup` of each resource using `kubectl api-resources`.
And, the `apiGroup` of `Role` is `rbac.authorization.k8s.io`.
```console
$ kubectl api-resources | grep -e NAME -e roles
NAME                              SHORTNAMES   APIVERSION                             NAMESPACED   KIND
clusterroles                                   rbac.authorization.k8s.io/v1           false        ClusterRole
roles                                          rbac.authorization.k8s.io/v1           true         Role
```

So, it is better to set `rbac.authorization.k8s.io` to `roleRef.apiGroup` if we set `Role` to the `roleRef.kind`.
(For some reason it seems to be working with `roleRef.apiGroup` is `""`... But I didn't make sense the reasons...)

---

By the way, when we set `ServiceAccount` to the `subjects.kind`, there is no problem with value of `subjects.apiGroup` is `""`.
Because the `apiGroup` of `ServiceAccount` is `core` and the `""` indicates the core API group.
```console
$ kubectl api-resources | grep -e NAME -e serviceaccount
NAME                              SHORTNAMES   APIVERSION                             NAMESPACED   KIND
serviceaccounts                   sa           v1                                     true         ServiceAccount
```
* Note: The `v1` means `core` API group.